python/tvm/relay/frontend/mxnet.py

```diff
@@ -1105,7 +1260,10 @@ def _from_mxnet_impl(symbol, shape_dict, dtype_info, mod=None):
             raise tvm.error.OpNotImplemented(
                 'Operator {} is not supported in frontend MXNet.'.format(op_name))
 
-    outputs = [node_map[e[0]][e[1]] for e in jgraph["heads"]]
+    outputs = []
```

Comment:
seems these lines should be equivalent to original code

Reply:
This is for debugging purposes, the original code is hard to insert debugging.